src/translations/en.json

```
@@ -92,6 +92,28 @@
         "title": "Select a card",
         "cannotMove": "Oops! It seems like you can't play. Select a Jack to scrap.",
         "resolve": "Resolve"
+      },
+      "statsScoringDialog": {
+        "ranksDetermination": "How are ranks determined?",
+        "competitiveCuttle": {
+          "description": "Competitive Cuttle is divided into 4 seasons per year, one per suit: Clubs, Diamonds, Hearts, and Spades. At the end of each season, the top 8 players play a double elimination championship tournament and the",
+          "highlight": "champions are permanantly accoladed on the site."
+        },
+        "seasonDescription": "Each Season is divided into 13 weeks. For each week, we count the number of best 2/3 ranked matches each player wins against unique opponents (ignoring stalemates) and assign players points based on their weekly standing.",
+        "points": {
+          "displayName": "Points",
+          "singular": "Point",
```

We can apply `i18n` pluralization to address both of these with a single key
```suggestion
          "displayName": "1 Point | {count} Points",
```